service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -248,26 +237,38 @@ public PipelineRun createPipelineRun(
             .addParameter(
                 RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_ID,
                 pipeline.getWorkspaceId().toString())
+            .addParameter(
+                RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_STORAGE_URL,
+                pipelineRun.getWorkspaceStorageUrl())
             .addParameter(
                 RunImputationJobFlightMapKeys.WDL_METHOD_NAME, pipeline.getWdlMethodName())
             .addParameter(JobMapKeys.RESULT_PATH.getKeyName(), resultPath);
 
     jobBuilder.submit();
 
-    logger.info("Created {} pipelineRun with jobId {}", pipelineName, jobId);
+    logger.info("Started {} pipelineRun with jobId {}", pipelineName, jobId);
 
     return pipelineRun;
   }
 
+  private Map<String, Object> retrievePipelineInputs(Long id) {
+    PipelineInput pipelineInput = pipelineInputsRepository.findById(id).orElseThrow();
```

Comment:
should we put something specific to throw here if the inputs dont exist in the database?

Reply:
sure - this really shouldn't happen - so we can throw an Internal Server Error with a somewhat descriptive message